playwright/e2e/osweb/test_subjects_footer.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import asyncio
+
+from e2e.ui.pages.home import HomeRex
+
+
+@pytest.mark.asyncio
+async def test_subjects_footer(chrome_page, base_url):
+
+    # GIVEN: Playwright, chromium and the rex_base_url
+
+    # WHEN: The Home page is fully loaded
+    await chrome_page.goto(base_url)
+    home = HomeRex(chrome_page)
+
+    await home.click_subjects_page_menu()
+
+    await home.click_subjects_homepage_link()
+
+    # THEN: Footer section loads
+    assert home.footer_section
+
+    assert home.footer_section_help_is_visible
+    assert home.footer_section_openstax_is_visible
+    assert home.footer_section_policies_is_visible
+
+    assert home.footer_section_bottom_is_visible
+
+    assert "Rice University" and "license" in await home.footer_section_bottom_is_visible.inner_text()
```

Comment:
The `in` operator has higher precedence than `and` so this is equivalent to `assert "Rice University" and ("license" in await home.footer_section_bottom_is_visible.inner_text())` so the "Rice University" part is not testing anything. So you either need 2 different asserts or use `in` twice, like: `"Rice University" in ... and "license" in ...`